gapic/templates/examples/feature_fragments.j2

```diff
@@ -0,0 +1,127 @@
+{#
+ # Copyright (C) 2019  Google LLC
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+ #}
+
+{#
+A careful reader may comment that there is duplication of effort
+between the python verification step and the dispatch/rendering here.
+There is a little, but not enough for it to be important because
+1) Other python artefacts (client libraries, unit tests, and so forth)
+   are generated using templates, so doing the same for generated samples is consistent.
+2) Using jinja for anything requiring real logic or data structures is a bad idea.
+#}
+
+{# response handling macros #}
+
+{% macro handlePrint(elts) %}
```

Comment:
Are these _handling_ or _rendering_?

Also, I did this with include files (see `$proto/_messages.py.j2`). Macros might be better...it does not need to be in this PR, but we should pick one approach and do that.

Reply:
I used _handling_ as a naming system because the dispatch is similar to the dispatch in the python code. Thoughtless naming convention on my part.
I would argue that because they take parameters, macros are a better solution both in terms of modularity and testing. Plus, doing the recursive dispatch using includes sounds like hell on wheels.

Reply:
Template macros now using the 'render' moniker to differentiate with the 'handle' moniker in python code.

Reply:
Yeah, I am sold that macros are better than includes. I will update my code to match what you have done here.